debezium-core/src/test/java/io/debezium/util/Testing.java

```diff
@@ -26,6 +26,8 @@
 import io.debezium.util.Stopwatch.Statistics;
 import io.debezium.util.Stopwatch.StopwatchSet;
 
+import javax.xml.ws.Holder;
```

Comment:
> Let's avoid importing this class. It will cause issues down the road when running on the Java Module System, where we wouldn't want to depend on the JAX-WS module.

Yes I agree, it's bad idea.